bindings/azure/storagequeues/storagequeues.go

```diff
@@ -70,13 +70,17 @@ func (d *AzureQueueHelper) Init(accountName string, accountKey string, queueName
 func (d *AzureQueueHelper) Write(data []byte, ttl *time.Duration) error {
 	ctx := context.TODO()
 	messagesURL := d.queueURL.NewMessagesURL()
-	s := string(data)
```

Comment:
Two questions:
1. Is this a breaking change?
2. Can the handling of byte[] be done via a Content-Type metadata instead?

Reply:
1. It could be a break change if you are removing the double quotation and you don't check if there aren't any quotation to remove. To avoid this, may be we can add a new option in metadata to indicate if the component must remove the double quotation, indicate this break change in the next release and then change the behaviour to remove always the double quotation

2. About Content-Type, I don't know how to do it. Can you give me more information?

Reply:
/cc @artursouza

Reply:
@artursouza can you check my comments?